### PR TITLE
remove Kubernetes

### PR DESCRIPTION
**Pull request by nicogbg**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Remove Kubernetes spec and kubernetes container  from CBL-Mariner

###### Change Log  

- remove the relevant files

###### Does this affect the toolchain?  

NO

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -2260,66 +2260,6 @@
...
-          "downloadUrl": "https://kubernetesartifacts.azureedge.net/kubernetes/v1.20.2-hotfix.20210525/binaries/kubernetes-node-linux-amd64.tar.gz"
-        }
-      }
-    },
-    {
-      "component": {
-        "type": "other",
-        "other": {
-          "name": "kubernetes",
-          "version": "1.20.5",
```
Comment (PawelWMS) [inline cgmanifest.json:2318]: Please also update `licenses.json` and re-build the licenses md file.

**Review by PawelWMS — CHANGES_REQUESTED** [review]

File: cgmanifest.json (hunk 1 above)
Comment (nicogbg) [reply cgmanifest.json]: as per off line discussion, latest version of k8s released should be kept in cgmanifest.json and licenses.json

**Review by PawelWMS — APPROVED** [review]

**Review by PawelWMS — CHANGES_REQUESTED** [review]
After some consideration, please also add update the licenses check defined inside `.github/workflows/check-license-map.yml` so that it doesn't crash for the Kubernetes specs. I'd suggest extending it by an exception list, so that we don't let any real issues slip through and keep the check valid.

**Comment by nicogbg** [comment]
> After some consideration, please also update the licenses check defined inside `.github/workflows/check-license-map.yml` so that it doesn't crash for the Kubernetes specs. I'd suggest extending it by an exception list, so that we don't let any real issues slip through and keep the check valid.

[nicogbg] fixed

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**